distrobox-init

```diff
@@ -391,8 +400,8 @@ if ! grep -q "${container_user_name}" /etc/passwd; then
 	fi
 fi
 # Delete password for root and user
-printf "password\npassword\n" | passwd "${container_user_name}"
-printf "password\npassword\n" | passwd root
+printf "password1234ADFG_+\npassword1234ADFG_+\n" | passwd "${container_user_name}"
```

Comment:
I was thinking it would be nice at this point if we use a random string for the password
We set it in a variable (so that it's possible to use it again for confirmation)

Maybe something like `cat /proc/sys/kernel/random/uuid`?

Reply:
Alternatively

`< /dev/urandom tr -dc '[:alpha:]' | head -c20`

Reply:
Added that. Whats the advantage of that?

Reply:
> Added that. Whats the advantage of that?

Just that we do not have the password in the repo this way and being just a temp one, we can (and should) use a random one at this point